app/views/users/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<%# <nav class="navbar bg-primary text-secondary">
+  <div class="custom-nav-container">
+    <button
+      class="nav-buttons bg-primary text-secondary"
+      type="button"
+      aria-label="Toggle navigation"
+    >
+      <i class="material-icons outlined">menu</i>
+    </button>
+      <h2 class="custom-navbar-header"><%= 'Home' %>
+      <%# </h2> %>
+  <%# </div> %>
+<%# </nav> %> 
+<%# <section class= "sign_in_holder">
+ <h1 class="main-page-header">Budget app tracker</h1>
+ <button class="btn_signin">
+  <%= link_to "Sign In", new_user_session_path %>
+ <%# </button> %>
+ <%# <button class="btn_signup"> %>
+  <%# <%= link_to "Sign Up", new_user_registration_path %> 
+ <%# </button> %>
+<%# </section> %> 
+
```

Comment:
- [ ] Please kindly delete the commented lines of code to make your codebase clean and professional 🙃